packages/clerk-js/src/ui/components/OrganizationSwitcher/InPlaceAcceptedInvitations.tsx

```diff
@@ -0,0 +1,47 @@
+import type { OrganizationResource, UserOrganizationInvitationResource } from '@clerk/types';
```

Comment:
This should be renamed and moved into the `ui/contexts` directory.

Reply:
Should we ? This is something that is needed only for OrganizationSwitcher